bedrock/settings/base.py

```diff
@@ -406,6 +406,19 @@ def lazy_langs():
 SESSION_COOKIE_SECURE = not DEBUG
 SESSION_ENGINE = 'django.contrib.sessions.backends.cache'
 
+# legacy setting. backward compat.
+DISABLE_SSL = config('DISABLE_SSL', default=True, cast=bool)
```

Comment:
We could do that now. I could just go ahead and add the new config where appropriate if you'd like, and remove this line. I just did this so that we wouldn't have to immediately need to change prod config.